### ISSUE TITLE
SSL routine wrong version number on gcloud beta emulators pubsub

### ISSUE THREAD

**Issue by lafama** [GoogleCloudPlatform/google-cloud-node#2580]
Error:
``E0831 07:46:08.107115000 140736173401024 ssl_transport_security.c:921] Handshake failed with fatal error SSL_ERROR_SSL: error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number.``
#### Environment details

  - OS:  darwin x64
  - Node.js version: 8.4.0
  - npm version: 5.3.0
  - google-cloud/pubsub version: 0.14.0
  - gcloud pubsub-emulator 2017.08.07
  - Google Cloud SDK 169.0.0

#### Steps to reproduce

  1. require google-cloud/pubsub
  2. topic=pubsub.topic('name)
  3. publisher= topic.publisher()
  4. publisher.publish(new Buffer('message'),cb)

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by mkamioner** [comment]
BUMP!

I am having the same issue after upgrading to `0.14.0`

```E0831 18:52:02.729735573   17326 ssl_transport_security.c:940] Handshake failed with fatal error SSL_ERROR_SSL: error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number.```

**Environment details**

- OS: Ubuntu 17.04
- Node Version: 6.11.2
- npm version: 3.10.10
- pubsub-emulator: 2016.08.19
- Google Cloud SDK: 130.0.0

**Steps to reproduce**
```
let client = require('@google-cloud/pubsub')(config);
client.createTopic('fooBar', (err) => console.log(err));
```

PS - Thanks @lafama for the idea to give the environment details and the steps to reproduce!

**Comment by callmehiphop** [comment]
@lafama @mkamioner sorry about that! I've made a PR with a potential fix (https://github.com/GoogleCloudPlatform/google-cloud-node/pull/2583), would either of you be so kind as to test it out for me?

**Comment by lafama** [comment]
@callmehiphop i have tried the new code. Still the same problem
```E0901 07:40:06.674602000 140736173401024 ssl_transport_security.c:921] Handshake failed with fatal error SSL_ERROR_SSL: error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number.```

**Comment by biggestT** [comment]
@callmehiphop your PR #2583 solved this issue for me. (just copied the two changed lines into my `node_modules/@google-cloud/pubsub/src/index.js`) Thanks, saved my day!

**Environment details**

OS: Arch Linux
Node Version: 6.11.1
npm version: 4.6.1
pubsub-emulator: not sure
Google Cloud SDK: 169.0.0

**Comment by callmehiphop** [comment]
@lafama could you prepare some code that reproduces this issue for me?

**Comment by lafama** [comment]
@callmehiphop i think i was not over-writing the file @biggestT  mentioned. Now it works
Thank you

**Comment by jasperkuperus** [comment]
Ok, PR #2583 fixes this for me too. When will it be published on npm?

**Comment by callmehiphop** [comment]
@jasperkuperus just published `0.14.1`
